### PR TITLE
Fixed #35887 -- Added imports and admin.site.register to non-partial admin inline doc examples.

### PR DESCRIPTION
**Pull request by e11bits**
#### Trac ticket number

ticket-35887

#### Branch description
There are some missing bits and pieces in the examples that make it harder than necessary to incorporate them.

#### Checklist
- [x] This PR targets the `main` branch. 
- [x] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [x] I have checked the "Has patch" ticket flag in the Trac system.
- [ ] I have added or updated relevant tests.
- [x] I have added or updated relevant docs, including release notes if applicable.
- [ ] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -2549,6 +2567,8 @@ so by defining an ``InlineModelAdmin`` object for the relationship::
 
     from django.contrib import admin
 
+    from .models import Group
```
Comment (sarahboyce) [inline docs/ref/contrib/admin/index.txt:2570]:
```suggestion
    from .models import Group, Person
```

Comment (sarahboyce) [reply docs/ref/contrib/admin/index.txt:2570]: Actually for this example specifically, I think it's worth us only defining `GroupAdmin` to keep this focussed on what it's trying to show so:
```
    from django.contrib import admin

    from .models import Group


    class MembershipInline(admin.TabularInline):
        model = Group.members.through


    class GroupAdmin(admin.ModelAdmin):
        inlines = [
            MembershipInline,
        ]
        exclude = ["members"]


    admin.site.register(Group, GroupAdmin)
```

File: docs/topics/db/multi-db.txt [hunk 2]
```diff
@@ -683,6 +683,13 @@ Once you've written your model admin definitions, they can be
 registered with any ``Admin`` instance::
 
     from django.contrib import admin
+    from .models import Author, Publisher
+
+
+    # insert class MultiDBModelAdmin(admin.ModelAdmin):
+
+
+    # insert class MultiDBTabularInline(admin.TabularInline):
```
Comment (sarahboyce) [inline docs/topics/db/multi-db.txt:692]: Perhaps similar to the widget example
```suggestion

    from .models import Author, Publisher
    # Import our custom ModelAdmin and TabularInline from where they're defined.
    from myproject.admin import MultiDBModelAdmin, MultiDBTabularInline
```

**Review by sarahboyce — COMMENTED** [review]
Thank you @e11bits 
I have small suggestions :+1:

File: docs/ref/contrib/admin/index.txt (hunk 1 above)
Comment (e11bits) [reply docs/ref/contrib/admin/index.txt]: True

**Comment by e11bits** [comment]
Thanks, I commited your suggestions!

**Review by sarahboyce — APPROVED** [review]
Thank you!

**Review by sarahboyce — APPROVED** [review]
Thank you for the updates @e11bits - I am happy with this

**Pull request merged**